frontend/agaya-gangai-explorer/style.css

```
@@ -0,0 +1,557 @@
+}
+
+.flow-img {
+    width: 100%;
+    height: 250px;
+    object-fit: cover;
+    border-radius: 8px;
+}
+
+.flow-desc h3 {
+    margin: 0 0 10px;
+    font-family: 'Outfit', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Outfit" (font-family-name-quotes)

(font-family-name-quotes)